deployment/deploy.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/bash
 
-DEPLOYMENT_USER=ubuntu
-DEPLOYMENT_HOST="testing.govstack.global"
-DEPLOYMENT_DIR="/opt/testing-webapp"
+# Check branch
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [ "${CURRENT_BRANCH}" == "main" ]; then
```

Comment:
I would prefer the branch to come from $1, so we would call this: 

deploy.sh staging
deploy.sh prod

I don't want this script to rely on git branches - we should keep that isolated to the circle config.